src/api/test/functional/channel_maintenance_test.rb

```diff
@@ -22,6 +22,10 @@ def setup
   # And it is doing a following up update, based on released updates
   #
   def test_large_channel_test
+    # https://github.com/rails/rails/issues/37270
```

Comment:
Please add a FIXME here so we find it later...

Reply:
done :)